std/socket.d

```diff
@@ -629,48 +629,18 @@ private mixin template socketOSExceptionCtors()
 
 
 /**
- * Class for exceptions thrown from an $(D InternetHost).
+ * Class for exceptions thrown from an `InternetHost`.
  */
 class HostException: SocketOSException
 {
     mixin socketOSExceptionCtors;
 }
 
 /**
- * $(D InternetHost) is a class for resolving IPv4 addresses.
+ * `InternetHost` is a class for resolving IPv4 addresses.
  *
- * Consider using $(D getAddress), $(D parseAddress) and $(D Address) methods
+ * Consider using `getAddress`, `parseAddress` and `Address` methods
  * instead of using this class directly.
- *
- * Example:
```

Comment:
Yes in general ddoced examples are horrible, because we can't ensure that they even compile.
As I seem not to be able to add a comment in the lines outside of the diff: maybe you could replace the commented `writefln` with `assert`'s, so that it won't look to weird to a reader?
Note that for the rendered documentation we are [considering](https://github.com/dlang/dlang.org/pull/1297) to replace `assert`'s like `a == b` automatically with a writeln equivalent